app/views/shelters/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>All Shelters</h1>
+
+<% @shelters.each do |shelter| %>
+    <h2><%= shelter.name %></h2>
+<% end %>
```

Comment:
Only shelter names are shown here, the rest of the shelter details are available at each shelter's view.